README.md

```diff
@@ -1,10 +1,21 @@
 # Overview
 
-This repo provides example JSON body sections to enable and disable the CIS Benchmarks for GCP in Lacework when calling the PATCH method on *https://lacework-tenant.lacework.net/api/v1/external/recommendations/gcp* API Endpoint
+This repo provides example JSON body sections to enable and disable the CIS Benchmarks for Azure in Lacework when calling the PATCH method on *https://lacework-tenant.lacework.net/api/v1/external/recommendations/azure* API Endpoint via the Lacework CLI
 
-# Script
-This repo also includes an example script that can be used to perform the config, with a requirement of having curl and the Lacework CLI installed.  Usage is:
+# Script - azure-cis-config.py
+This script can be used to bulk enable/disable compliance checkers on your target environment.
 
-`gcp-cis-config.sh [disable_cis_10|disable_cis_12|enable_cis_12] [lacework-tenant]`
+### Pre-requisites
+- Lacework CLI installed
+- Lacework CLI configured, targeting the desired Lacework tenant
```

Comment:
can you put a quick note/link to how to set it up ... worked example ... lens here is to ensure a CS or SE eng can as quickly as possible run this and fix up and env

Reply:
maybe call out awk and other deps ?